include/voxen/util/elapsed_timer.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <voxen/config.hpp>
+
+#include <fmt/format.h>
+
+namespace voxen
+{
+
+/**
+ * This is class is voxen-specific class for code segment measurements.
+ * The class will collect time point on constructor/stop method and print them into log (Debug level) automatically.
+ * Should be used like this:
+ *    ElapsedTimer test_timer("section name");
+ *    <measurable code here>
+ *    test_timer.stop()
+ */ 
+class ElapsedTimer 
+{
+public:
+    ElapsedTimer(std::string section_name, std::string format = "[Elapsed timer] execution of section \"{}\" tooks {} ms");
+    ElapsedTimer();
+    ElapsedTimer(const ElapsedTimer& other) = delete;
+    ElapsedTimer(ElapsedTimer&& other) = delete;
+    ~ElapsedTimer();
+
+    void stop();
+
+private:
+    std::string m_section_name;
+    std::string m_format;
+    bool m_finished;
```

Comment:
Please use `... = value;` initializer for `m_finished` (and other variables with default values) instead of doing it in constructors